### PR TITLE
Use SSL for requests

### PR DESCRIPTION
**Pull request by lagr**

### CODE REVIEW

**Comment by JanBe** [comment]
Fix qualification 🎓

**Comment by lagr** [comment]
I need to update the specs

File: lib/firmenwissen/http_request.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@ def initialize(uri, options = {})
     end
 
     def execute
-      http = Net::HTTP.start(uri.host)
+      http = Net::HTTP.start(uri.host, { use_ssl: true })
```
Comment (timoschilling) [inline lib/firmenwissen/http_request.rb]: `http = Net::HTTP.start(uri.host, use_ssl: true)`

Comment (lagr) [reply lib/firmenwissen/http_request.rb]: Yes, thats cleaner.

File: spec/http_request_spec.rb [hunk 2]
```diff
@@ -17,7 +17,7 @@
     let(:http_get_mock) { double(basic_auth: true) }
 
     before do
-      allow(Net::HTTP).to receive(:start).with(uri.host).and_return(http_mock)
+      allow(Net::HTTP).to receive(:start).with(uri.host, { use_ssl: true}).and_return(http_mock)
```
Comment (timoschilling) [inline spec/http_request_spec.rb]: same here

**Review by timoschilling — APPROVED** [review]

File: spec/http_request_spec.rb (hunk 2 above)
Comment (lagr) [reply spec/http_request_spec.rb]: I think this slipped when running rubocop...
image

**Pull request merged**